eng/ilverify.bsl

```diff
@@ -0,0 +1,555 @@
+[IL]: Error [StackUnexpected]: : Internal.Utilities.Collections.HashMultiMap`2::.ctor(int32, [S.P.CoreLib]System.Collections.Generic.IEqualityComparer`1<!0>, [FSharp.Core]Microsoft.FSharp.Core.FSharpOption`1<bool>)][offset 0x00000060][found ref 'object'][expected ref '[S.P.CoreLib]System.Collections.Generic.IDictionary`2<T0,Microsoft.FSharp.Collections.FSharpList`1<T1>>'] Unexpected type on the stack.
```

Comment:
These are all issues we have currently in `FSharp.Core` and `FSharp.Compiler.Service` when building with `Proto` compiler.